src/config.py

```diff
@@ -273,15 +288,18 @@ def scram_properties(self) -> List[str]:
         Returns:
             list of scram properties to be set
         """
+        username = INTER_BROKER_USER
+        password = self.internal_user_credentials.get(INTER_BROKER_USER, "")
```

Comment:
**review note:** Setting these to default to empty does mean that during cluster init, the leader will load credentials without passwords, but they will be immediately updated by subsequent events after user credentials have been added to ZooKeeper. 

These events **should** fire before listeners open up anyway so is probably fine.